packages/terra-form-validation/src/FormValidationUtil.js

```diff
@@ -0,0 +1,80 @@
+/* eslint-disable no-underscore-dangle */
+/* eslint-disable no-restricted-globals */
+
+class FormValidationUtil {
+  /* Checks given value's length against maximum. Returns true if value's length is less than the maximum.
+   * @value is the value string, @maximum is a nonnegative whole number. */
+  static isUnderMaxLength(value, maximum) {
+    if (!(typeof value === 'string' || value instanceof String)
+    || !(typeof maximum === 'number' && isFinite(maximum))
+    || maximum < 0 || !Number.isInteger(maximum)) { return false; }
+    return value.length <= maximum;
+  }
+
+  /* Checks given value's length against minimum. Returns true if value's length is greater than the minimum.
+   * @value is the value string, @minimum is a nonnegative whole number. */
+  static isOverMinLength(value, minimum) {
+    if (!(typeof value === 'string' || value instanceof String)
+    || !(typeof minimum === 'number' && isFinite(minimum))
+    || minimum < 0 || !Number.isInteger(minimum)) { return false; }
+    return value.length >= minimum;
```

Comment:
Should this be > instead of >= based on the function description?

Reply:
Same deal with the maximum comment. Meant for the minimum to be an acceptable value, fixed description here https://github.com/cerner/terra-framework/pull/635/commits/415c46ab9a0b74893a686c38430b1d8decd708ad#diff-efb0b388a04429df36f1ce4eb79ef872R108